### ISSUE TITLE
Fix Dockerfile not building

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/scala-test-runner#6]
Trying to build the Docker image using `docker build -t exercism/scala-test-runner .` fails with the following error:

```
------
 > [ 7/12] COPY dependencies/ivy2.tar.gz /opt/scala-test-runner:
------
------
 > [ 8/12] COPY dependencies/sbt.tar.gz /opt/scala-test-runner:
------
failed to compute cache key: "/dependencies/sbt.tar.gz" not found: not found
```

**Comment by MeerKatDev** [comment]
I just noticed that it's something connected to containerisation, and I have no experience in using Docker with SBT/Scala, so I wouldn't know how to solve this.

**Comment by ErikSchierboom** [comment]
Fixed in #7
